cli/src/cluster_query.rs

```diff
@@ -336,10 +342,10 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         "Only show stake accounts delegated to the provided vote accounts. "),
                 )
                 .arg(
-                    Arg::with_name("lamports")
-                        .long("lamports")
-                        .takes_value(false)
-                        .help("Display balance in lamports instead of SOL"),
+                    pubkey!(Arg::with_name("withdrawer_pubkey")
```

Comment:
nit: Maybe this is just me, but why not call this arg `withdraw_authority` or `withdraw_authority_pubkey`? (and same for the variables that use this value)